app/models/v2/policy.rb

```diff
@@ -46,5 +48,19 @@ def profile_title
     def ref_id
       attributes['profile__ref_id'] || profile.ref_id
     end
+
+    def update_relationships(new_host_ids, _user)
+      # Remove all hosts assigned to policy then import the new ones
+      # FIXME: scope this after V2::Host is implemented
+      removed = V2::PolicyHost.where(policy: self)&.destroy_all
```

Comment:
```suggestion
      removed = self.policy_hosts.destroy_all
```

Reply:
Also here the `delete_all` could make sense as we don't need to validate the deletions and we want this to be fast as it is customer facing.